### PR TITLE
Feat/sprint3

### PR DESCRIPTION
**Pull request by PaulBOURGEAIS**

### CODE REVIEW

File: src/service/survey-service.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import { lunaticDatabase } from "service/lunatic-database";
+import { getCurrentPageSource } from "service/orchestrator-service";
+
+const datas = new Map<string, LunaticData>();
+const activitySurveysIds = ["activitySurvey1", "activitySurvey2", "activitySurvey3"];
+const workingTimeSurveysIds = ["workingSurvey1", "workingSurvey2"];
+const surveysIds = [...activitySurveysIds, ...workingTimeSurveysIds];
+
+const enum FieldNameEnum {
+    lastName = "LASTNAME",
```
Comment (MichaelC67) [inline src/service/survey-service.ts:12]: Use uppercase ?
https://google.github.io/styleguide/jsguide.html#features-objects-enums
(the same for L13/L14)

**Review by MichaelC67 — CHANGES_REQUESTED** [review]

File: src/pages/help/Help.tsx [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
     return (
         <>
-            <header>Help - {t("home-page.welcome")}</header>
+            <header>Help - {t("home.page.welcome")}</header>
+
+            <ClickableList
+                options={activites}
+                handleChange={() => console.log("handleChange")}
+                createActivity={() => console.log("createActivity")}
+                placeholder="Saisissez une activité"
```
[the remark this answers is not in the document]
Comment (PaulBOURGEAIS) [reply src/pages/help/Help.tsx:16]: Code here is temporar, it's just for tests purpose

File: src/pages/work-time/weekly-planner/WeeklyPlanner.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            onNavigateBack={navBack}
+            firstName={getPrintedFirstName(context.idSurvey)}
+            surveyDate={getPrintedSurveyDate(context.idSurvey)}
+        >
+            <FlexCenter>
+                <OrchestratorForStories
+                    source={context.source}
+                    data={context.data}
+                    callbackHolder={callbackHolder}
+                    page="3"
```
[the remark this answers is not in the document]
Comment (PaulBOURGEAIS) [reply src/pages/work-time/weekly-planner/WeeklyPlanner.tsx:62]: The weekly planner is page 3 of the work-time-survey.json

**Review by MichaelC67 — APPROVED** [review]

**Comment by MichaelC67** [comment]
Waiting the CI/CD to be fixed to merge if SonarCLoud analysis is ok.

**Review by MichaelC67 — APPROVED** [review]

**Pull request merged**